tests/test_cli_args.py

```
@@ -1,7 +1,217 @@
+        # This is ambiguous - separator at position 1 means no profile
+        try:
+            separator_idx = argv.index('--')
+            if separator_idx == 1:
+                # No synapse args, everything after -- is tool args
+                synapse_args = []
+                tool_args = argv[separator_idx + 1:]
+            else:
+                synapse_args = argv[2:separator_idx]
+                tool_args = argv[separator_idx + 1:]
+        except ValueError:
+            synapse_args = argv[2:]
```

Ruff (0.14.10)

Local variable `synapse_args` is assigned to but never used

Remove assignment to unused variable `synapse_args`

(F841)